src/esp/assets/ResourceManager.cpp

```diff
@@ -2910,6 +2910,12 @@ void ResourceManager::createDrawable(Mn::GL::Mesh* mesh,
                              : nullptr);  // pbr image based lighting
       break;
   }
+
+  drawableCountAndNumFaces_.first += 1;
+  if (mesh) {
+    drawableCountAndNumFaces_.second += mesh->count() / 3;
```

Comment:
Best would be to query this on the `Trade::MeshData` instance before a `GL::Mesh` is made out of it (it's available at the point `createDrawable()` is called, as far as I see), there you get both index and vertex count, total data size etc.

For robust calculation of primitive count that does the right thing for triangle strips/fans as well there's [MeshTools::primitiveCount()](https://doc.magnum.graphics/magnum/namespaceMagnum_1_1MeshTools.html#a746d76c4b365565d5f796d2ca74dc8bc) (meant to be called with `meshData.primitive()`, not the GL mesh primitive).